### PR TITLE
[tests] Fix bgen test to compile by adding missing using.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: tests/generator/BGenTests.cs [hunk 1]
```diff
@@ -336,7 +336,6 @@ public void StrongDictSupportTemplatedDicts ()
 		}
 
 		[Test]
-		[Ignore ("Fails with: api.cs(10,17): error CS0246: The type or namespace name `CBUUID' could not be found. Are you missing `CoreBluetooth' using directive?")]
```
Comment (spouliot) [inline tests/generator/BGenTests.cs]: looks like an unrelated change / not mentioned in the commit message

Comment (rolfbjarne) [reply tests/generator/BGenTests.cs]: Nope, it's on purpose, this is the test that's being fixed. Since it's fixed, we don't have to ignore it anymore.
